addons/survey/models/survey_user_input.py

```diff
@@ -187,6 +198,22 @@ def action_print_answers(self):
             'url': '/survey/print/%s?answer_token=%s' % (self.survey_id.access_token, self.access_token)
         }
 
+    def action_redirect_to_attempts(self):
+        self.ensure_one()
+
+        action = self.env['ir.actions.act_window']._for_xml_id('survey.action_survey_user_input')
+        context = dict(self.env.context or {})
+
+        context['search_default_survey_id'] = self.survey_id.id
+        context['search_default_group_by_survey'] = False
+        if self.partner_id:
+            context['search_default_partner_id'] = self.partner_id.id
+        elif self.email:
+            context['search_default_email'] = self.email
+
+        action['context'] = context
```

Comment:
Shouldn't we say create=false while being here ? Seems more a to-check list than a to-create.